src/plugin/inputmethod_p.h

```diff
@@ -177,12 +177,11 @@ class InputMethodPrivate
 
         if (QGuiApplication::platformName() == "ubuntumirclient") {
             view->setFlags(InputMethodWindowType); /* Mir-only OSK window type */
```

Comment:
Do we have a convention for single-line comments? `//` or `/* */`? Or do we just not care?

Reply:
I think most of this component uses `//` and plenty of multi-line comments use that too. But also this code is pre-existing and not part of this change set, so we shouldn't change it here.

Reply:
right